src/widgets/ModalBar.js

```diff
@@ -239,21 +239,10 @@ define(function (require, exports, module) {
         }
     };
     
-    /**
-     * If autoClose is set, detects when something other than the modal bar is getting focus and
-     * dismisses the modal bar. DOM nodes with "attached-to" jQuery metadata referencing an element
-     * within the ModalBar are allowed to take focus without closing it.
-     */
     ModalBar.prototype._handleFocusChange = function (e) {
```

Comment:
This is a generic modal bar focus handler and may not be the right place to do this change. 
What you might want to do is to find out the specific modal bar for find/replace [see src/search/findbar.js ] and create a isLockedOpen() function to lock focus handling.